evalai/utils/challenges.py

```diff
@@ -21,17 +24,21 @@ def print_challenge_table(challenge):
     title = "{} {}".format(challenge_title, challenge_id)
 
     description = "{}\n".format(challenge["short_description"])
-    date = "End Date : " + style(challenge["end_date"].split("T")[0], fg="red")
-    date = "\n{}\n\n".format(style(date, bold=True))
-    challenge = "{}{}{}{}".format(title, description, date, br)
+    end_date = "End Date : " + style(challenge["end_date"].split("T")[0], fg="red")
+    end_date = "\n{}\n\n".format(style(end_date, bold=True))
+    challenge = "{}{}{}{}".format(title, description, end_date, br)
     echo(challenge)
 
 
-def get_challenges(url):
+def display_challenges(url):
+
```

Comment:
Any reason for this extra line here?

Reply:
No 😄. Thanks for the catch. I'll fix it and merge the PR.